examples/js/ThreeLoader.js

```diff
@@ -0,0 +1,71 @@
+/* global itowns, Promise */
+var manager = new itowns.THREE.LoadingManager();
+
+/**
+ * Load a script asynchronously
+ * @param {String} uri URI of the script to load.
```

Comment:
the documentation is generated?

Reply:
since it's in a namespace, it's not in the output of the generated doc when I add it in jsdoc-config.json